src/main/resources/config/application-gcp.yml

```diff
@@ -0,0 +1,50 @@
+# ===================================================================
+# Spring Boot configuration for the "gcp" profile.
+#
+# This configuration overrides the application.yml file.
+#
+# More information on profiles: https://www.jhipster.tech/profiles/
+# More information on configuration properties: https://www.jhipster.tech/common-application-properties/
+# ===================================================================
+
+# ===================================================================
+# Standard Spring Boot properties.
+# Full reference is available at:
+# http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
+# ===================================================================
+
+spring:
+  cloud:
+    gcp:
+      sql:
+        enabled: true
+        database-name:
+        instance-connection-name:
+  datasource:
+    hikari:
+      data-source-properties:
+        socketFactory: com.google.cloud.sql.mysql.SocketFactory
+        cloudSqlInstance:
+      jdbcUrl:
+      username:
+      password:
```

Comment:
unless these override a value provided in base application.yml or other profile-specific yml (prod/dev), I will prefer to not duplicate here.

Reply:
Thanks. I've removed the duplicate properties that are present in `application-gcp.yml`. However properties such as `jdbcUrl` are not defined in `application-prod.yml` or `application.yml` so I think I have to keep them there for the substitutions. :smile: :thinking: